src/main/java/com/stablekernel/rxespresso/RxIdlingResource.java

```diff
@@ -93,13 +98,12 @@ public void registerIdleTransitionCallback(ResourceCallback resourceCallback) {
     @Override
     public <T> Observable.OnSubscribe<T> onSubscribeStart(Observable<? extends T> observableInstance,
                                                           final Observable.OnSubscribe<T> onSubscribe) {
+
         int activeSubscriptionCount = subscriptions.incrementAndGet();
+        onSubscribeList.put(new WeakReference<Observable.OnSubscribe>(onSubscribe), new Throwable());
```

Comment:
and in reality, it should never go away before we're done with it since we're tracking it during its lifecycle.  so doing this just makes it a little harder to work with.